README.md

```diff
@@ -103,3 +91,7 @@ Examples:
 
 Report any issue to this
 [Github project](https://github.com/ursais/odoo-template/issues).
+
+## Roadmap
+
+* Push the production database to the other environments
```

Comment:
@JevinD That is done. Not in the roadmap anymore...

Reply:
@max3903 I updated with my next objective for documentation. Is there any other plans to include?